src/common/utils/useBackendSorting.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+import { usePrevious } from './usePrevious';
+
+interface SortBy {
+  id: string;
+  desc?: boolean;
+}
+
+export const useBackendSorting = (onOrderByChange?: Function) => {
+  const [orderBy, setOrderBy] = useState<string>();
+
+  // sortBy: the id of the column to be sorted in the table
+  // orderBy: the value sent to the backend for sorted results
```

Comment:
This naming is at least a little confusing

Reply:
For the lack of imagination 😄 
I'll try to come up with better names; suggestions are welcome though!

Reply:
Renamed!